python/cuml/test/test_kmeans.py

```diff
@@ -48,6 +45,15 @@ def test_n_init_cluster_consistency(random_state):
                       cluster_std=cluster_std,
                       shuffle=False,
                       random_state=0)
+    return X, y
+
+
+@pytest.mark.xfail
+@pytest.mark.parametrize('random_state', [i for i in range(10, 2)])
```

Comment:
shouldn't this be `range(2, 10)`? Otherwise this is an empty list. 
```suggestion
@pytest.mark.parametrize('random_state', [i for i in range(2, 10)])
```